lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -3477,47 +3517,40 @@ def test_defaults(self, mock_set_custom_attribute):
             vertical = ItemFactory.create(category='vertical', parent_location=subsection.location)
             ItemFactory.create(category='problem', parent_location=vertical.location, has_score=True)
 
-        with patch('lms.djangoapps.courseware.views.views.get_enrollment') as mock_get_enrollment:
```

Comment:
This was a result of you cleaning up the unneeded get_enrollment import in the views file, right?

Reply:
That's correct. Without this change, the test will fail with an error complaining cannot patch a missing import reference.